src/OrchardCore.Build/Dependencies.props

```diff
@@ -47,7 +47,7 @@
     <PackageManagement Include="NodaTime" Version="3.1.4" />
     <PackageManagement Include="OpenIddict.AspNetCore" Version="3.1.1" />
     <PackageManagement Include="OpenIddict.Core" Version="3.1.1" />
-    <PackageManagement Include="OrchardCore.Translations.All" Version="1.4.0" />
+    <PackageManagement Include="OrchardCore.Translations.All" Version="1.0.1-beta-20024" />
```

Comment:
Why it's `1.0.1-beta`?!!

Reply:
crowdin only, just incrementing the build number